src/Illuminate/Cache/MemcachedStore.php

```diff
@@ -42,8 +42,12 @@ public function __construct($memcached, $prefix = '')
         $this->setPrefix($prefix);
         $this->memcached = $memcached;
 
-        $this->onVersionThree = (new ReflectionMethod('Memcached', 'getMulti'))
-                            ->getNumberOfParameters() == 2;
+        try {
+            $this->onVersionThree = (new ReflectionMethod('Memcached', 'getMulti'))
+                                ->getNumberOfParameters() == 2;
+        } catch (\Exception $e) {
```

Comment:
There are implementations of Memcached out there, particularly on Windows, where Memcached is not well supported, in which the implementation has no getMulti method. So, the attempt to find out how many parameters getMulti has throws an exception and kills Laravel.

Memcached::getMulti is only called from MemcachedStore::many, so if one never invokes that later method, Laravel is perfect usable on a system in which Memcached does not support getMulti ... as long as one can make it though the constructor.

Hope that made sense.

I can resubmit using with a different detection method if you want (say use ReflectionClass' hasMethod to ensure the method exists before asking how many parameters it has).

I submitted this against 5.3 because that's the version we're using right now, 5.3 is still being supported and new releases overwrite our modification to this class.